Added a genric borderize function and added text in requested table view

gnana997 opened the pull request:
#8 
Hi @jonas-grgt , I have created a generic borderize function in Styles.go which can be used to place text in between anywhere in table border and used it in:
- Topic List View
- Consumer Group List View
- Consumer Subjects List View

I would like some feedback from you if everything is good or should I make some changes?

Thanks

jonas-grgt commented:
That PR dropped faster than I expected! Thanks for the effort.

I do think it needs some more work. 

Now we have a double border one from the table and one from the actual new border you've introduced. 
It looks a bit clunky, so I would get rid of the outer table border.
Also something is not quite right with the width calculation as you can see in my screenshot.

<img width="973" alt="Screenshot 2025-03-09 at 20 25 23" src="https://github.com/user-attachments/assets/9216b3cf-72d8-4d4b-99e2-9b9f8b6daac1" />

gnana997 commented:
Hi @jonas-grgt , I have updated the borderize function and also removed some default border styles in `styles.go`. Can you please review and let me know?

gnana997 commented:
Hi @jonas-grgt , I will take a look at the test cases. I might need to update some test cases. Can you please test it once and let me know if everything looks good? I will update the test cases accordingly.

jonas-grgt commented:
@gnana997 tried the branch out and noticed that the table border is now gone everywhere. 

For example when consuming the list of records does not have a (table) border anymore, same goes for the subjects page.

Else everything is looking really nice!

gnana997 commented:
Hi @jonas-grgt , I have updated the test cases to include the embedded text in the table border. It is passing all the test cases in m machine. can you please review and let me know?
Thanks

gnana997 commented:
Hi @jonas-grgt , Can you test them again once? I have refactored all the other components using table. All the tests are also passing as expected. Let me know. If you find something is missing
Thanks

jonas-grgt commented:
Looking good, can you solve the conflict so I can rebase and merge it into main?

jonas-grgt commented:
I've merged it into main. It will be part of the 0.3.0 release. Thank you very much for your efforts and time!
